src/main/java/org/gitlab4j/api/SnippetsApi.java

```diff
@@ -0,0 +1,125 @@
+package org.gitlab4j.api;
+
+import java.util.List;
+import javax.ws.rs.core.GenericType;
+import javax.ws.rs.core.Response;
+import org.gitlab4j.api.models.Snippet;
+import org.gitlab4j.api.models.Visibility;
+
+/**
+ * This class provides an entry point to all the GitLab Snippets API project calls.
+ */
+public class SnippetsApi extends AbstractApi {
+
+	public SnippetsApi(GitLabApi gitLabApi) {
+		super(gitLabApi);
+	}
+
+    /**
+     * Create a new Snippet.
+     *
+     * @param title the title of the snippet
+     * @param fileName the file name of the snippet
+     * @param content the content of the snippet
+     * @return the created Snippet
+     * @throws GitLabApiException if any exception occurs
+     */
+	public Snippet createSnippet(String title, String fileName, String content) throws GitLabApiException {
+		GitLabApiForm formData = new GitLabApiForm()
+				.withParam("title", title)
+				.withParam("file_name", fileName)
+				.withParam("content", content);
```

Comment:
The setting ```title```, ```file_name```, and ```content``` must have a boolean third paramter specifying that they are required per the docs at: https://docs.gitlab.com/ee/api/snippets.html#create-new-snippet

Should look like this:
```
GitLabApiForm formData = new GitLabApiForm()
    .withParam("title", title, true)
    .withParam("file_name", fileName, true)
    .withParam("content", content, true);
```